docs/user/package-management/repo-management.md

```diff
@@ -3,103 +3,36 @@ title: Repository Management
+| ---------------- | -------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------- |
+| Stable (shannon) | `https://cdn.getsol.us/repo/shannon/eopkg-index.xml.xz`  | Default software source.                                                                                    |
+| Unstable         | `https://cdn.getsol.us/repo/unstable/eopkg-index.xml.xz` | Only use this source if build Solus packages or you test new software before it moves to the stable source. |
 
 :::danger
 
-It is not always safe to perform a system update when using the **unstable** repository. Always check the topic of the [#solus-packaging](https://matrix.to/#/#solus-packaging:matrix.org) Matrix room before updating.
+Updating your system while using the **unstable** source might be risky.
 
-:::
-
-## Adding a repository
-
-You can add a repository by using:
-
-```bash
-sudo eopkg add-repo Name URL
-```
-
-For example:
-
-```bash
-sudo eopkg add-repo Example https://example.com/repo/eopkg-index.xml.xz
-```
-
-**Note:** This does not enable the repository.
-
-## Removing a repository
-
-You can remove a repository by using:
-
-```bash
-sudo eopkg remove-repo Name
-```
-
-For example:
-
-```bash
-sudo eopkg remove-repo Example
-```
-
-## Enabling a repository
-
-You can enable a repository for usage, by using:
+Before updating, check the [#solus-packaging](https://matrix.to/#/#solus-packaging:matrix.org) Matrix room.
 
-```bash
-sudo eopkg enable-repo Name
-```
-
-For example:
-
-```bash
-sudo eopkg enable-repo Example
-```
-
-## Disabling a repository
-
-You can disable a repository by using:
-
-```bash
-sudo eopkg disable-repo Name
-```
-
-For example:
-
-```bash
-sudo eopkg disable-repo Example
-```
-
-## Updating a repository
-
-You can update the index of a repository by using:
-
-```bash
-sudo eopkg update-repo Name
-```
-
-For example:
-
-```bash
-sudo eopkg update-repo Example
-```
-
-**Note:** You can update all repositories by **not** providing a name.
+:::
 
-## List repositories
+## Repository management commands
 
-You can list all the repositories added on your system by using:
+The following table lists the `eopkg` commands you can use to manage your software sources.
 
-```bash
-eopkg list-repo
-```
+| Action                                 | Command                        | Example                                                                   | Notes                                                    |
+| -------------------------------------- | ------------------------------ | ------------------------------------------------------------------------- | -------------------------------------------------------- |
+| Add a repository                       | `sudo eopkg add-repo Name URL` | `sudo eopkg add-repo Example https://example.com/repo/eopkg-index.xml.xz` | This doesn't enable the repository.                      |
```

Comment:
The note here is no longer true; adding a repository does automatically enable it.

Reply:
Updated